### ISSUE TITLE
video content

### ISSUE THREAD

**Issue by xsaxasni** [ethz-asl/vgn#33]
Hello, your work is really great. 
After watching your paper presentation video, I have a little question. At the end of the video, the camera does not scan according to a fixed trajectory, but remains motionless. In this case, it can also show good modeling effects and grasping assumptions. Is this achieved by taking only one depth map to obtain the TSDF?
Looking forward to your reply.

**Comment by mbreyer** [comment]
Yes, for the last clip, the input TSDF of the network was created from a single depth map. The network was trained on TSDFs created with random number of viewpoints, so it still handles the single view case, though from experience grasping performance does increase with more complete reconstructions.

**Comment by xsaxasni** [comment]
Thank you for your reply!
